src/app/ruler/ruler.css

```diff
@@ -76,17 +76,37 @@ body {
 }
 
 .ruler__inner__info {
+	background-color: rgba(34,34,34,0.3);
 	position: absolute;
-	bottom: 1.2em;
-	left: 5.6em;
-	width: 6.5em;
+	bottom: .6em;
+	left: 4.9em;
+	padding: 3px 5px 3px 3px;
 	font-size: 1.1em;
 	color: white;
+	z-index: 99;
+}
+
+@media (max-width:152px) {
+	.ruler__inner__info {
+		left: 1em;
+	}
+}
+
+.dark-theme .ruler__inner__info {
+	background-color: rgba(255,255,255,0.7);
+}
+
+.ruler__inner__info__width:before,
+.ruler__inner__info__height:before {
+	width:18px;
+	text-align: center;
```

Comment:
Adding the width and align center here just tricks the eye to make the `W:` and `H:` feel mono-spaced (for OCD eyes like mine).